app/controllers/users/registrations_controller.rb

```diff
@@ -0,0 +1,20 @@
+module Users
+  class RegistrationsController < Devise::RegistrationsController
+    respond_to :json
+
+    private
+
+    def respond_with(resource, _opts = {})
+      if resource.persisted?
+        render json: {
+          status: { code: 200, message: 'Signed up sucessfully.' },
+          data: resource
+        }
+      else
```

Comment:
* Hi, i get an error when i try to test your restful endpoints please kindly fix this 
![image](https://user-images.githubusercontent.com/52497246/140453881-ef263d77-2d6a-4526-b00f-8d2ea6e58320.png)
![image](https://user-images.githubusercontent.com/52497246/140453901-f62ffb3e-ea95-48c1-8e7d-b77c39542677.png)
![image](https://user-images.githubusercontent.com/52497246/140453911-bc9af440-50aa-48b1-9474-bba503285191.png)

Reply:
- Appears to be a working find on the contrary when we test from our own end. But we will have a look again and see if we can spot the problem. 
Thank you for the feedback 🤝

Reply:
Is very important to add in the request the correct headers the content type as we mentioned in the Readme file { "Accept": "application/json", "Content-Type": "application/json" }, because another way is not going to work, in your request you used type Text, please change that type to JSON and tell us if is it workig
![test](https://user-images.githubusercontent.com/60273425/140520569-2a49539d-5ac5-4568-87c4-baf154b3c822.png)


![Create_user](https://user-images.githubusercontent.com/60273425/140521074-6d960542-3412-4c77-9513-c41e16ce28a4.PNG)

Reply:
- Here is another test response using the thunderbolt one test case that was made correctly with the JSON request Header and another one with text header and it failed as in your case so please kindly check to reconfirm. Thank you 🤝 
   - [x] *** This screenshot of JSON header and response***
>![Screen Shot 2021-11-05 at 3 23 32 PM](https://user-images.githubusercontent.com/50230297/140526300-1d106d8a-827e-4300-9590-759acb0ed54a.png)

   - [x] *** This screenshot of Text header and response ***
>![Screen Shot 2021-11-05 at 3 24 46 PM](https://user-images.githubusercontent.com/50230297/140526311-92b36280-0dab-466a-9f27-d45038a7a263.png)